.github/workflows/consuming.yml

```diff
@@ -53,6 +53,9 @@ jobs:
           repository: submariner-io/${{ matrix.project }}
           path: ${{ matrix.project }}
 
+      - name: Copy Dockerfile.dapper if necessary
+        run: test -f ${{ matrix.project }}/Dockerfile.dapper || cp Dockerfile.dapper ${{ matrix.project }}/
+
       - name: Make sure ${{ matrix.project }} is using the built Shipyard image
         run: sed -i 's/shipyard-dapper-base:*.*/shipyard-dapper-base:dev/' ${{ matrix.project }}/Dockerfile.dapper
```

Comment:
Just a question - why not have it as part of this step?

It seems closely tied, I'd put them both in the same step

Reply:
I was looking at fixing #491 too, with a more generic step; does it make better sense like this?

Reply:
Hmm yeah makes sense I guess.

Though what would make even more sense IMHO is to extract these all repetitive steps to it's own composite action and just call that, but it's a bit outside the scope of this specific PR :)

Reply:
Yup, a specific consuming composite action would be nice ;-).